runelite-api/src/main/java/net/runelite/api/IndexDataBase.java

```diff
@@ -29,4 +29,8 @@
  */
 public interface IndexDataBase
 {
+	/**
+	 * Returns true if any cache overlay is outdated due to hash mismatch
```

Comment:
`Returns true if any cache overlay in this index  is outdated due to hash mismatch`

Reply:
Updated